source/mqtt-tls.rst

```diff
@@ -0,0 +1,341 @@
+Using MQTT with security (TLS)
+==============================
+
+.. note::
+   - Audience: users
+   - Level: intermediate
+   - Reading time: 15 m
+
+This document describes how to configure the dojot and
+publish/subscribe using MQTT over TLS (MQTTS)
+when using the microservice Broker MQTT
+`IotAgent-VerneMQ`_ or Legacy Broker MQTT `IotAgent-Mosca`_.
+
+.. contents:: Table of Contents
+  :local:
+
+
+
+Components
+----------
+
+X.509 Identity Management
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The purpose of this component is to provide x.509 identification for
+final dojot entities, that is, IoT devices that communicate with the
+dojot IoT platform. See more about in `X.509 Identity Management`_.
+
+What is a certificate?
+^^^^^^^^^^^^^^^^^^^^^^
+
+A certificate contains the public key for an entity (a user, device, website),
+along with information about this entity, about the CA which signs the
+certificate, the allowed certificate usage and a checksum. When a entity wants
+a certificate to be signed, the entity should create a CSR file and send it to
+the desired CA. The CSR file is an 'intention of certification'. The file
+contains the information required from the entity and some information about
+the certificate use, hostnames and IPs where the certificate will reside,
+alternative names for the entity, etc.
+
+Brokers MQTT
+~~~~~~~~~~~~
+
+Broker MQTT IotAgent VerneMQ (Default)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The IotAgent VerneMQ extends `VerneMQ`_ with some features and services
+for dojot case, see more in `IotAgent-VerneMQ`_.
+This is the **default** Broker MQTT in dojot deployments.
+
+To use IotAgent VerneMQ with TLS, you need to configure by environment variable:
+
+ - EXTERNAL_SERVER_HOSTNAME: Server hostname/ip (the host to which the device connects externally).
+   Example: mydomain.com. By default is *localhost*.
+
+See others environment variables in `VerneMQ Dojot - Environment variables`_.
+
+Broker MQTT IotAgent Mosca (Legacy)
```

Comment:
Sounds a bit strange to me, maybe only `Mosca (legacy)` as a title would do the job.